gsoc_contributors_guide.md

```diff
@@ -43,74 +42,55 @@ of ocean sonar data.
 
 The Dataset has the dimensions and coordinates: 
 
-* frequency - transducer frequency used for the experiment, with units Hz
-
-* ping_time - timestamp of each ping
-
-* range_bin - sample index along a range
-
-The data variables of the Dataset are as follows, where items in parenthesis are the dimensions of the data variables: 
-
-* `Sv` (frequency, ping_time, range_bin) - volume backscattering strength measured from the echo
-
-* `range` (frequency, ping_time, range_bin) - the measured range of an echo in meters 
-
-* `temperature` - the temperature measurement of the water collected by the echosounder, with unit degree Celsius
-
-* `salinity` - the salinity measurement of the water collected by the echosounder, with unit part per thousand (PSU)
-
-* `pressure` - the pressure measurement of the water collected by the echosounder, with unit dbars
+* `frequency` - sonar transducer frequency, with units Hz
+* `ping_time` - timestamp of each ping
+* `range_bin` - sample index along a range
 
-* `sound_speed` (frequency, ping_time) - sound speed (in units m/s) for the provided temperature, salinity, and pressure 
+The data variables of the Dataset are listed below, where items in parenthesis are the dimensions of the data variables: 
 
-* `sound_absorption` (frequency, ping_time) - sea water absorption (in units dB/m) for each frequency and ping time, this 
-value is based on the temperature, salinity, pressure, and sound_speed
+* Key data variables you will be working with:
+    * `Sv` (frequency, ping_time, range_bin) - volume backscattering strength measured from the echo
+    * `range` (frequency, ping_time, range_bin) - the measured range of an echo in meters 
+* Other variables included in this dataset so that the parameters used in the calibration (from raw to Sv) are recorded:
```

Comment:
```suggestion
* Other variables included in this dataset. These are included so that the exact parameters used in the calibration (from raw to Sv) are recorded:
```

Reply:
Done